src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -636,6 +639,14 @@ def _get_protected_item_instance(item_type):
 
 
 def _check_map(item_type, item_type_map):
+    if item_type is None:
+        if item_type_map == workload_type_map:
+            raise RequiredArgumentMissingError("Workload type missing. Please enter a valid workload type.")
```

Comment:
To make the message more actionable, we may also tell users which argument they can use to pass a valid workload type